pages.zh/linux/xcowsay.md

```diff
@@ -0,0 +1,29 @@
+# xcowsay
+
+> 在您的 Linux 桌面上显示一头可爱的牛和指定的消息。
+> 牛的显示时间是固定的或则是根据文本大小计算得出的。 点击牛即马上关闭。
+> 更多信息： <https://www.doof.me.uk/xcowsay/>.
+
+- 显示一头说 “hello, world” 的牛：
+
+`xcowsay "{{hello, world}}"`
+
+- 显示一头有另一个命令的输出的牛：
+
+`ls | xcowsay`
+
+- 显示一头有指定 X 和 Y 坐标的牛：
```

Comment:
```suggestion
- 显示一头牛，并且其位置由 X 和 Y 坐标指定：
```

Reply:
I thought the second sounded more natural, but the first was closer to the [original text](https://github.com/tldr-pages/tldr/blob/main/pages/linux/xcowsay.md).

Reply:
How about

```suggestion
- 显示一头指定 X 和 Y 坐标的牛：
```

Reply:
The original sentence is fine.